scala3-scala2-library-tasty-tests/test/Scala3Scala2LibraryTASYyTest.scala

```diff
@@ -0,0 +1,206 @@
+package dotty.tools.dotc
+
+import org.junit.Test
+import org.junit.Ignore
+import org.junit.Assert._
+
+import dotty.tools.io._
+import dotty.tools.dotc.util.ClasspathFromClassloader
+
+import scala.quoted._
+
+import java.io.File.pathSeparator
+
+class Scala3Scala2LibraryTASYyTest:
+
+  import Scala3Scala2LibraryTASYyTest._
+
+  /** Test that we can load trees from TASTy */
+  @Test def testTastyInspector: Unit =
+    loadWithTastyInspector(loadBlacklisted)
+
+  /** Test that we can load and compile trees from TASTy */
+  @Test def testFromTasty: Unit =
+    compileFromTasty(loadBlacklisted.union(compileBlacklisted))
+
+  @Ignore
+  @Test def testWhiteListFromTasty: Unit =
+    val whitelist = Set(
+      "scala.collection.mutable.StringBuilder"
+    )
+    compileFromTasty(x => !whitelist(x))
+
+  @Test def blacklistNoDuplicates =
+    def testDup(name: String, list: List[String], set: Set[String]) =
+      assert(list.size == set.size,
+        list.diff(set.toSeq).mkString(s"`$name` has duplicate entries:\n  ", "\n  ", "\n\n"))
+    testDup("loadBlacklist", loadBlacklist, loadBlacklisted)
+    testDup("compileBlacklist", compileBlacklist, compileBlacklisted)
+
+  @Test def blacklistsNoIntersection =
+    val intersection = loadBlacklisted & compileBlacklisted
+    assert(intersection.isEmpty,
+      intersection.mkString(
+        "`compileBlacklist` contains names that are already in `compileBlacklist`: \n  ", "\n  ", "\n\n"))
```

Comment:
That error message mentions "compileBlacklist" twice when it probably should mention "loadBlacklist"?

Reply:
Now that the blacklist has shrunk I we can do without checking it.

Reply:
indeed